src/Integration/Monolog/CloudWatch/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Max Leonov
```

Comment:
revert

Reply:
is the code different enough to not require citing the source?

https://github.com/maxbanton/cwh/blob/master/LICENSE